examples/braket_features/Verbatim_Compilation.ipynb

```diff
@@ -316,24 +316,13 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "DECLARE ro BIT[1]\n",
-      "PRAGMA INITIAL_REWIRING \"NAIVE\"\n",
-      "RX(pi) 0\n",
-      "RX(pi) 0\n",
-      "MEASURE 0 ro[0]\n",
-      "\n",
-      "Counter({'0': 1000})\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
+    "# set up the Rigetti Aspen-M-1 device\n",
```

Comment:
We should rely on the rigetti object made in a latter cell and not include this line.